src/Utils/isArtsyEmail.ts

```diff
@@ -1,3 +1,7 @@
-export const isArtsyEmail = (email: string) => {
+export const isArtsyEmail = (email: string | null | undefined): boolean => {
+  if (!email) {
+    return false
+  }
+
   return /@artsymail.com$/.test(email)
```

Comment:
```tsx
return email && /@artsymail.com$/.test(email)
```

Reply:
Just want to clarify the intention here as this will return `falsy` values and so we could end up returning `null`, `underfined` or `""`. I would think we would want to always return a boolean.

Or do we just want it to be a one-liner? In which case I can switch to a ternary:

```JS
return email ? /@artsymail.com$/.test(email) : false
```

Reply:
Oh, sorry, I just want it to not be something like: if (true) { return false } — which I find deeply weird to read.

```tsx
return !!email && /@artsymail.com$/.test(email)
```